tests/foreman/ui/test_org.py

```diff
@@ -75,24 +104,20 @@ def test_positive_create_with_parent(self, parent_name):
         """
         org_name = gen_string('alpha')
         with Session(self.browser) as session:
-            make_org(session, org_name=parent_name)
-            make_org(
-                session,
-                org_name=org_name,
-                label=gen_string('alpha'),
-                desc=gen_string('alpha'),
-                parent_org=parent_name
-            )
-            self.assertIsNotNone(self.org.search(org_name))
+            for parent_name in generate_strings_list():
```

Comment:
@omaciel this is a stubbed test. Parent org feature is not available currently. We can worry about this when the feature is available :)